application/features/user.feature

```diff
@@ -433,7 +433,7 @@ Feature: User
       And the user store is empty
       And I provide the following valid data:
         | property     | value                 |
-        | employee_id  | 123-456               |
+        | employee_id  | 123&456               |
```

Comment:
I think it would be better to leave out the `employee_id` on this line (since this step is "I provide the following valid data:") and instead add a new line after this table (around line 441) that's something like this:
```
But I provide an invalid employee_id of "123&456"
```

Reply:
:+1: